improver/utilities/mathematical_operations.py

```diff
@@ -436,18 +436,21 @@ def verify_units_match(
         diagnostic_cube: Cube, mean_cube: Cube, std_cube: Optional[Cube] = None
     ) -> None:
         """Check that all cubes have the same units. E.g. to prevent accidental
-        use of cubes with rate data with cubes with accumulation data."""
+        use of cubes with rate data with cubes with accumulation data.
+
+        Raises:
+            ValueError: If the units of the cubes do not match.
+        """
         errors = []
         if mean_cube.units != diagnostic_cube.units:
             errors.append(
-                f"The mean cube must have the same units as the diagnostic cube."
+                "The mean cube must have the same units as the diagnostic cube. "
                 f"The following units were found: {mean_cube.units},"
                 f"{diagnostic_cube.units}"
             )
         if std_cube and std_cube.units != str(diagnostic_cube.units):
             errors.append(
-                f"The standard deviation cube must have the same units "
-                "as the diagnostic cube."
+                "The standard deviation cube must have the same units. "
```

Comment:
```suggestion
                "The standard deviation cube must have the same units as the diagnostic cube. "
```
I think that this was fine before.